src/pybind/mgr/dashboard/frontend/package-lock.json

```diff
@@ -3313,18 +3317,18 @@
       }
     },
     "apache-crypt": {
-      "version": "1.2.1",
-      "resolved": "https://registry.npmjs.org/apache-crypt/-/apache-crypt-1.2.1.tgz",
-      "integrity": "sha1-1vxyqm0n2ZyVqU/RiNcx7v/6Zjw=",
+      "version": "1.2.4",
+      "resolved": "https://registry.npmjs.org/apache-crypt/-/apache-crypt-1.2.4.tgz",
+      "integrity": "sha512-Icze5ny5W5uv3xgMgl8U+iGmRCC0iIDrb2PVPuRBtL3Zy1Y5TMewXP1Vtc4r5X9eNNBEk7KYPu0Qby9m/PmcHg==",
       "dev": true,
       "requires": {
-        "unix-crypt-td-js": "^1.0.0"
+        "unix-crypt-td-js": "^1.1.4"
       }
     },
     "apache-md5": {
-      "version": "1.1.2",
-      "resolved": "https://registry.npmjs.org/apache-md5/-/apache-md5-1.1.2.tgz",
-      "integrity": "sha1-7klza2ObTxCLbp5ibG2pkwa0FpI=",
+      "version": "1.1.5",
```

Comment:
Just wondering, why is nearly any package upgraded?

Reply:
when you run `npm uninstall`, it also updates dep packages.
Since we haven't update packages for some time, there was a lot of changes.
I have manually removed `ngx-store`, so the changes are small now.